item.rb

```diff
@@ -0,0 +1,36 @@
+require 'date'
+
+class Item
+  attr_reader :label, :genre, :author, :source, :publish_date, :archived, :id
+
+  def initialize(id, publish_date)
+    @id = id
+    @publish_date = publish_date
+    @archived = false
+  end
+
+  def add_genre(genre)
+    @genre = genre
+  end
+
+  def add_author(author)
+    @author = author
+  end
+
+  def label=(label)
+    @label = label.title
+    label.items << self unless label.items.include?(self)
+  end

```

Comment:
- [ ] in the `add_genre` & `add_author` methods you should have included the following piece of code to ensure the 1 to many relationships work correctly.
`genre.items.push(self) unless genre.items.include?(self)`
`author.items.push(self) unless author.items.include?(self)`
- [ ] There should be an equal sign between the method name and the arguments like there is for the label.

Reply:
@ichala @mandelatuks @kwambiee It would be better if you add the piece of codes 
- `genre.items.push(self) unless genre.items.include?(self)`
- `author.items.push(self) unless author.items.include?(self)`
without these your methods it's not making the 1 to many relationships but with them it will ensure the 1 to many relationships so please I encourage you to do that